src/routers/user.js

```diff
@@ -0,0 +1,53 @@
+const express = require('express');
+
+const routes = express.Router();
+
+const { check } = require('express-validator');
+const UserController = require('../controllers/user');
+
+routes.post(
+  'login',
+  check('username', 'Username must be at least 4 characters long')
+    .notEmpty()
+    .withMessage('Username should not be empty')
+    .bail(),
+  check('password', 'Password must be at least 5 characters long')
+    .notEmpty()
+    .withMessage('Password should not be empty')
+    .bail(),
+  UserController.login
+);
+
+routes.post(
+  'register',
```

Comment:
here also '/register' since its coming from /users but there it will be /usersregister